server/src/main/java/org/apache/seata/server/filter/XSSHttpRequestFilter.java

```diff
@@ -59,11 +65,26 @@ public class XSSHttpRequestFilter implements HttpRequestFilter {
             Pattern.CASE_INSENSITIVE);
 
     public XSSHttpRequestFilter() {
-        String xssKeywordConfig = CONFIG.getConfig(SERVER_HTTP_FILTER_XSS_FILTER_KEYWORDS, DEFAULT_XSS_KEYWORDS);
+        String xssKeywordConfig = CONFIG.getConfig(SERVER_HTTP_FILTER_XSS_FILTER_KEYWORDS, null);
+
+        if (StringUtils.isBlank(xssKeywordConfig)) {
```

Comment:
If the user does not configure SERVER_HTTP_FILTER_XSS_FILTER_KEYWORDS, then do not enable XSSHttpRequestFilter?